multicluster/service-mirror/cluster_watcher.go

```diff
@@ -1141,17 +1165,7 @@ func (rcsw *RemoteClusterServiceWatcher) updateReadiness(endpoints *corev1.Endpo
 	}
 }
 
-func isExportedEndpoints(obj interface{}, log *logging.Entry) bool {
-	ep, ok := obj.(*corev1.Endpoints)
-	if !ok {
-		log.Errorf("error processing endpoints object: got %#v, expected *corev1.Endpoints", ep)
-		return false
-	}
-
-	if _, found := ep.Labels[consts.DefaultExportedServiceSelector]; !found {
-		log.Debugf("skipped processing endpoints object %s/%s: missing %s label", ep.Namespace, ep.Name, consts.DefaultExportedServiceSelector)
-		return false
-	}
-
-	return true
+func isExportedEndpoints(ep *corev1.Endpoints) bool {
+	_, found := ep.Labels[consts.DefaultExportedServiceSelector]
```

Comment:
Looks like this was a bug too, this is using the default label selector instead of the selector from the link (like isExportedService does)

Reply:
ah good find :+1: